applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/settings.js

```diff
@@ -0,0 +1,217 @@
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';	 
+
+	o = s.option(form.Value, 'min_snr_kick_delay', _('min_snr_kick_delay'), _('Timeout after which a station with snr < min_snr will be kicked'));
+	o.optional    = true;
+	o.placeholder = 5000;
+	o.datatype    = 'uinteger';	 
+
+	o = s.option(form.Value, 'roam_process_timeout', _('roam_process_timeout'), _('Timeout (in ms) after which a association following a disassociation is not seen as a roam'));
+	o.optional    = true;
+	o.placeholder = 5000;
+	o.datatype    = 'uinteger';	 
+	
+	o = s.option(form.Value, 'roam_scan_snr', _('roam_scan_snr'), _('Minimum signal-to-noise ratio or signal level (dBm) before attempting to trigger client scans for roam'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';	 
+
+
+	o = s.option(form.Value, 'roam_scan_tries', _('roam_scan_tries'), _('Maximum number of client roaming scan trigger attempts'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';	 
+
+	o = s.option(form.Value, 'roam_scan_timeout', _('roam_scan_timeout'), _('Retry scanning when roam_scan_tries is exceeded after this timeout (in ms). In case this option is set to 0, the client is kicked instead'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';	 
+
+	o = s.option(form.Value, 'roam_scan_interval', _('roam_scan_interval'), _('Minimum time (ms) between client roaming scan trigger attempts'));
+	o.optional    = true;
+	o.placeholder = 10000;
+	o.datatype    = 'uinteger';	 
+
+	o = s.option(form.Value, 'roam_trigger_snr', _('roam_trigger_snr'), _('Minimum signal-to-noise ratio or signal level (dBm) before attempting to trigger forced client roaming'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';	 
+
+	o = s.option(form.Value, 'roam_trigger_interval', _('roam_trigger_interval'), _('Minimum time (ms) between client roaming trigger attempts'));
+	o.optional    = true;
+	o.placeholder = 60000;
+	o.datatype    = 'uinteger';	
+
+	o = s.option(form.Value, 'roam_kick_delay', _('roam_kick_delay'), _('Timeout (in 100ms beacon intervals) for client roam requests'));
+	o.optional    = true;
+	o.placeholder = 100;
+	o.datatype    = 'uinteger';	
+
+	o = s.option(form.Value, 'signal_diff_threshold', _('signal_diff_threshold'), _('Minimum signal strength difference until AP steering policy is active'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';	
+
+	o = s.option(form.Value, 'initial_connect_delay', _('initial_connect_delay'), _('Initial delay (ms) before responding to probe requests (to allow other APs to see packets as well)'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';	
+
+	o = s.option(form.Value, 'load_kick_enabled', _('load_kick_enabled'), _('Enable kicking client on excessive channel load (0/1)'));
```

Comment:
Should this be a flag (instead of "value") ?

Reply:
I was not quite sure what the behavior would be, by default it is not in the config file. With a flag, you can only do yes/no, not "leave default"? any advice?

Reply:
As the value is optional, and as the global default of optional flags is 0 = disabled, any value of '0' will be removed from config file by LuCI, when you edit settings. Only if you tick the box, there will be `option load_kick_enabled '1'` stored in the config file. 
(Having a value placeholder 0 is the same as having the flag tickbox empty...)

Reply:
but what happens if somebody want to force it off, regardless of the default (the default may change room version to version you know)?

Reply:
> the default may change room version to version you know)?

The you have to change also the default in LuCI, so that LuCI defaults match the app defaults.

Reply:
true but then it still requires an action of the user, even if the app is update.  Anyway, if we do not really care about this, then sure, i can make it into a flag. Let me do this